core/client/src/main/java/alluxio/client/file/options/CreateFileOptions.java

```diff
@@ -90,6 +93,14 @@ public long getTtl() {
   }
 
   /**
+   * @return the {@link TtlExpiryAction}; It informs the action to take when Ttl is expired. It can
```

Comment:
Are you indicating not to have get method for TtlAction.
Sorry, I didn't understand the comment. Can you elaborate?